docs/docs/other-css-frameworks.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Other CSS Frameworks
+---
+
+This guide introduces some other popular styling libraries that you might want to use in your project.
+
+## How these frameworks were chosen
+
+The following criteria were used to determine which frameworks to include in this guide:
+
+- It must work with React and Gatsby.
+- It must impact the styling of a Gatsby site.
+- It should be a commonly requested or searched in the Gatsby docs.
+- It must consider accessibility and not promote UI anti-patterns.
+
+## Bootstrap
+
+Bootstrap is a widely used CSS and JS framework that was created at Twitter in 2010. It gained popularity for its support of responsive and mobile-first design.
+
+Under the hood, Bootstrap uses [Sass](https://sass-lang.com/). Unlike other frameworks in this guide, Bootstrap does not provide React components directly. Instead, it provides a set of pre-built CSS class names like `"btn-primary"` or `"dropdown"` that you can use to style HTML elements.
```

Comment:
I think you an do without the quotes here! Let me know if you saw this pattern elsewhere in the docs?

```suggestion
Under the hood, Bootstrap uses [Sass](https://sass-lang.com/). Unlike other frameworks in this guide, Bootstrap does not provide React components directly. Instead, it provides a set of pre-built CSS class names like `btn-primary` or `dropdown` that you can use to style HTML elements.
```